google-cloud-speech/lib/google/cloud/speech/v1/speech_client_config.json

```diff
@@ -1,40 +1,40 @@
 {
   "interfaces": {
-    "google.cloud.speech.v1beta1.Speech": {
+    "google.cloud.speech.v1.Speech": {
       "retry_codes": {
-        "idempotent": [
-          "DEADLINE_EXCEEDED",
-          "UNAVAILABLE"
-        ],
-        "non_idempotent": [
-          "UNAVAILABLE"
-        ]
+          "idempotent": [
```

Comment:
Can you point me at documentation for how the idempotent and non_idempotent configuration works?  

I was wondering why UNAVAILABLE appears on both idempotent and non_idempotent list. There are other potential error codes where retries would be appropriate, e.g. INTERNAL.  The API is pretty much stateless, so retrying is always safe.  (Retrying calls might result in duplicated work but it won't mutate state twice or out of order.)